content/en/synthetics/guide/_index.md

```diff
@@ -1,38 +1,53 @@
 ---
 title: Synthetic Monitoring Guides
 kind: guide
+aliases:
+  - /synthetics/faq/clone-test/
+  - /synthetics/faq/uptime-check-internal-website/ 
+  - /synthetics/guide/browser-tests-switch-tabs/
+  - /synthetics/guide/browser-test-self-maintenance/
```

Comment:
I think you should redirect these to the page where the info moved (browser tests advanced) instead of this guides landing page.